CMakeLists.txt

```diff
@@ -240,6 +242,9 @@ if(CLR_CMAKE_PLATFORM_XPLAT)
         # -Wno-return-type # switch unreachable code
         # -Wno-switch # switch values not handled
 
+    add_compile_options(-fexceptions)
+    add_compile_options(-fstack-protector-all)
```

Comment:
Ok... when we are ready to look at perf, this is an area to measure to see if it actually make a difference or not.